### PR TITLE
reformatted application forms

### PR DESCRIPTION
**Pull request by BryceSlade**
[Loom Video](https://www.loom.com/share/cead7e3de9444b9aba202886057828f5)

## Contributors
@jcpcab 

## Description

This branch involved small changes to the application forms for the mentor and mentee. Now, all required fields share the same styling and all have the required star next to the label. Additionally, I cleaned up some of the styling to line up the form items and make it look more cohesive. This makes for a much cleaner application page. 

## Type of change

- [X] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] This change requires a documentation update

## Checklist:

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [X] I have removed unnecessary comments/console logs from my code
- [ ] I have made corresponding changes to the documentation if necessary (optional)
- [X] My changes generate no new warnings
- [X] I have checked my code and corrected any misspellings
- [X] No duplicate code left within changed files
- [X] Size of pull request kept to a minimum
- [X] Pull request description clearly describes changes made & motivations for said changes

### CODE REVIEW

**Comment by BryceSlade** [comment]
May need the UX team to go over these changes and determine what they are looking for. The wireframes show another style and it could have gone one of two ways. Keep the red required star on all forms, changing the style of the whole page, or keeping the text star in the heading tags (form validation will still work).

**Review by tarasshulhan — APPROVED** [review]

**Review by morgankj — APPROVED** [review]

File: src/components/pages/RoleSignup/Applications/Mentor.js [hunk 1]
```diff
@@ -146,30 +145,46 @@ const Mentor = () => {
...
                     onChange={handleChange}
-                    style={{ margin: '.5rem 1rem 1rem' }}
+                    style={{ margin: '0.5rem 1rem 1rem 0' }}
                   >
                     <Input placeholder="Enter Valid Email" />
                   </Form.Item>
                 </Col>
 
-                <Col span={24}>
+                {/* <Col span={24}>
```
Comment (fletchulence) [inline src/components/pages/RoleSignup/Applications/Mentor.js:154]: get rid of the comment

**Review by fletchulence — CHANGES_REQUESTED** [review]
change the small things for formatting

File: src/components/pages/RoleSignup/Applications/Mentee.js [hunk 2]
```diff
@@ -267,45 +263,54 @@ const Mentee = () => {
...
-                    >
-                      From a lower socioeconomic background
-                    </Checkbox>
-                    <Checkbox
-                      value="underrepresented_group"
-                      onChange={e => handleChange(e, 'checkbox')}
-                      style={{ margin: '.2rem', width: '100%' }}
+                    <Checkbox.Group
+                      label="Which criteria represents you for membership? (Select all
+                      that apply)"
```
[the remark this answers is not in the document]
Comment (BryceSlade) [reply src/components/pages/RoleSignup/Applications/Mentee.js]: This has been fixed. Thank you for catching that!

File: src/components/pages/RoleSignup/Applications/Mentor.js (hunk 1 above)
Comment (BryceSlade) [reply src/components/pages/RoleSignup/Applications/Mentor.js]: Comments removed!

**Comment by BryceSlade** [comment]
> 

Yes, these are from other components from different teams

File: src/components/pages/RoleSignup/Applications/Mentee.js [hunk 3]
```diff
@@ -1,4 +1,4 @@
-import React, { useState } from 'react';
+import React from 'react';
```
Comment (paulstgermain) [inline src/components/pages/RoleSignup/Applications/Mentee.js:1]: Good eye for cleaning up potential Linter warnings here.

**Review by paulstgermain — APPROVED** [review]
Thorough, consistent use of Ant Design components for our forms throughout - even introducing them in areas that weren't using them before - plus some clearly well-thought-out styling changes (Incremental progress, maybe, but vital to the flow of these forms!) and both a solid description and Loom walkthrough equals an easy pass from me.

Great work!

**Pull request merged**